osu.Game/Beatmaps/Formats/LegacyStoryboardDecoder.cs

```diff
@@ -64,15 +64,16 @@ protected override void ParseLine(Storyboard storyboard, Section section, string
         private void handleEvents(string line)
         {
             var depth = 0;
-            var lineSpan = line.AsSpan();
 
-            while (lineSpan.StartsWith(" ", StringComparison.Ordinal) || lineSpan.StartsWith("_", StringComparison.Ordinal))
+            foreach (char c in line)
             {
-                lineSpan = lineSpan.Slice(1);
-                ++depth;
+                if (c == ' ' || c == '_')
+                    depth++;
+                else
+                    break;
             }
 
-            line = lineSpan.ToString();
```

Comment:
`TrimStart(' ', '_')`

Reply:
have you benchmarked this? it's all much of a muchness because this isn't even a hot path, but probably worth knowing for future reference.

Reply:
No, but I believe the `TrimStart<T>(this ReadOnlySpan<T> span, ReadOnlySpan<T> trimElements) where T : IEquatable<T>` overload should have a good performance.

Reply:
I benchmarked it, allocations went from 418.15 MB to 436.69 MB, so even worse than before this PR.

Reply:
You can store the param array into a static field, which won't increase allocation.

Reply:
True, but I think the code looks cleaner this way.